### DISCUSSION TITLE
How to change logrotate.conf

### DISCUSSION THREAD

**Discussion by Greyb4ck** [telekom-security/tpotce#1227] [Q&A] [answered]
Hey, if I want to change my Log Persistence from 30 days to 90 for example, do I need to just change rotate in logrotate.conf like this?
{
	su tpot tpot
        copytruncate
        create 770 tpot tpot
	daily
        missingok
        notifempty
	rotate 30 -> to rotate 90
	compress
        compresscmd /usr/bin/pigz
}

/data/adbhoney/downloads.tgz
/data/cowrie/log/ttylogs.tgz
/data/cowrie/downloads.tgz
/data/dionaea/bistreams.tgz
/data/dionaea/binaries.tgz
/data/honeytrap/attacks.tgz
/data/honeytrap/downloads.tgz
{
	su tpot tpot
        copytruncate
        create 770 tpot tpot
	daily
        missingok
        notifempty
	rotate 30 -> to rotate 90
}

**Comment by t3chn0m4g3** [comment, marked as answer]
Yes.
